src/components/SQForm/SQFormIconButton.tsx

```diff
@@ -14,7 +14,7 @@ interface SQFormIconButtonProps {
   /** Type of button, defaults to 'submit' */
   type?: string;
   /** Standard React event handler */
-  onClick?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onClick?: (...event: React.MouseEvent<HTMLButtonElement>[]) => void;
```

Comment:
To my knowledge, there's only 1 argument (the `event`) on button `onClick` handlers.
I would expect something like this:

```javascript
onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
```